### DISCUSSION TITLE
Anchor / Bookmark Support

### DISCUSSION THREAD

**Discussion by freb** [GitbookIO/community#367] [Feature Requests]
It should be possible to link to a specific location within a page/document. This feature is hugely important for creating an index or reference table. Currently it is only possible to link to headings in a document.

Ideally the anchor or bookmark location could be named for easy discovery later (such as is done in word or with an HTML `id` attribute). Good Docs supports bookmarks but makes you select the target using surrounding text.

My specific use case is for creating a reference table in our compliance manual. We cannot move our manual to Gitbook because heading links are not granular enough. Our auditors require a reference to the text which satisfies each requirement.

In my custom markdown implementation I'm repurposing standard links as follows:

~~~markdown
[Invisble text of anchor](+bookmarkname)
[Displayed text of link](#bookmarkname)
~~~

But there have been [other syntax proposals](https://talk.commonmark.org/t/anchors-in-markdown/247).

**Comment by sarahf16** [comment]
This would be really useful for us too. To get around this I have added annotations with the information from each section I am linking to instead but this is very time-consuming and not the best solution.
